### DISCUSSION TITLE
Tiny Contribution for HDUse

### DISCUSSION THREAD

**Discussion by ArmanCevi** [Purfview/IMDb-Scout-Mod#142] [Ideas (implemented)]
Paid Usenet forum HDUse can be searched with the below settings
`  {   'name': 'HDUse',
      'icon': 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAHVUlEQVR42o2WA5ArSxuGn56eOJtVjv5r27Zt27Zt2ypd27Zt21xGyySDnv67KrOVnLNz8VY99fV0quZ9v+7pmVj8V615SCcbnLA+6x13jlj32CdjGxz3dXKjE3qSG5/UG9/whG/Fesc9bX67gPWO3Zg1Du7mP0qw4cn8i6aSym0FYme0WoZAdRgEgQ9KgTboABAghEZYI6Z+jg4exK09DPT9c4CV94Jo2XTOvjGJ3EnEUyuhdYxAYYwNobGprXOG1jAK+JjAv4zx0hOAGx1glb2JUBtt048hkTmaeLqTWApkbJLZfwvDiJm7EWf8EmAoIsCkFciRyZ9HLH2w6TxuKqZCLAkQYRRlbGjdokD5pt6GVzsJKM8cYMXdaFGCTPfZpuPjDbbpHlMbAcwYy4YgMPhRXf/ztVYKsxKMlU4Gqs0A6xxFi/bEkjdgJ7NMmMbCajDzgI66ebRxONesQR3lHAfcSCgbrQi1IDJxEkJksSSRSBtDhEmEmY78LUkQPw63+hbwBYAAGnX94y9E65ONUdjxxN63jkMg8mGM7lpNDqPcG/jhpaMAJVjrcID5jcmzBP782Ekilr85NoGEZZMWHmk9TuC7jCtJPZCgI05JVCjf+Yvx0ibAVzZ2AmBtdDAvwppl2e2Q8FpI2pOSrRZIsHS66onRcnVkeCjoG/HS3zgdiU/q0xhTcdARz4LyDC54CrSeHcvesBHg65dtFt9gLXRgIVuN5aQwuVjAQfOOBHus2PFrJjHHlzqYreAr3ymXSunfe/qW/qhUW/L2P7vsQj3e0rXBdyevhtZr8v2bN9li2iJ5HahFEQJEa7czdy4si9VTPayW7/y6PZt6JSbliAYPcNrbO6qzzT7nJ4uOjG6Z+jHY4OIPPeH5HigHXBWxNQblL2RPmXealZ8x21QLPR0hIrueqF2iyhLJUmWuOWd/OWbbva7n9ZRKpTcNbzmO02PbsnfalO571p7D+mPurAIEBEHEC6qBRZDv7Oz4nzUl39VpCZGM3vfmXF6MMSOX+DWfz/+klCoMDAy8DrwGvFOpVN4PgqAg4Fu30v/JdMr/fBwNltaJdDbTZcdsGRNCWIjosz8xn0lIOnJtFdu2B13XHR8eHu43QQhVMgGyiWTCqZQKg7omIUg3u49CK8soZlXHxmoBQkUte2sQL95BKteZU74/LKWsTJs2LfXEE0+I559/XqRSKduoPDQ0NFqsDE8tO4AOoo9gONbKV369WrX7B4rFIJ4dRcpuJkzF5OegR3fitsUXkrY9uyXEq11dXfbee++dU0o52WxWvf7aq0Vg1d5xsUqPys3a/aStCAJVHR0eKVijxd5BLeVfiKi9bwYo+3Ee/kXkvvzpz8N++/23OYQQhQUXXHAEqGNYeJHFFk20589/rzpt+rBOQeBF7v1EGB2o3uFysddm7iUqwKfGZPVZTRG2QYAOQDk8/bNFrVxadq1s383Jt9+5X2r1YWdnp55ngYWWoW36zo/1OQu/PRaAdiCIPH5NhPhC//VZwaahl43hAYbmaRBW4wa+07iJZeOZ+RcGs3xZnDr/QlKcOiPpeZkgwwtVL/ZFvSq+LiVRQfXfP8868A0vmRegkgC05UukOtbBTsyBFWt0jf7bb36NJH/obvGF0y2/rnfJP7w24WgbSyuceh2UC74b1hDlNGugvsSrXgiMShqqkp/HxpIbobVEz5TWoBoon2Vmy3LJVgugXJe+oRq20MRQzNNhs+vSXSw9I8U3PRVcx2matoZRXmC4AngewGJCIwMP4lZfNtCkRuu1reoctNpsbL74FDZauINT1pudm7abn+0W7+D9nwZ5+ot+jlprNtaYOwf+3/43eAu3ejehBJucRotWIZa+B0vOHXUULWmz5BzdLDxbJxqLQAiklPxUqNEz6rP3qnOx2rydHHnfZ/zSVwKv0USz1npxxvYAXmkG2OgkZtHOyMR1xi0/UwAhzVCiMWiLVDpJMhknk0oye2eGeafmcJTmwz9H+bM8jvbC1QsDGPMKzvhxvHzlbYAmlE3lR2bRA+TmiWHFLjGOMyZeTJ1tGXZeazHGXcVvxTGqpnbmMiRiMWpuQEpq6p7PgvkEMe3xc/9o8xlS3hBj5fP59tn7ATFzgA8eYRZp4ukHWWILl7apZxJLLoKQQirJz3/0MuYoLCmpVT3i+Eyf0c34WI0f/6rjBmDbNvVaDYKgYT7SX+enN97itw8K6GAJoAcoAzVACybLBrqAGeTnXZI5l92JtmlrYCdzIJqvadmolowRCCuct0GGOGPw+4c+P75WYbTwB/A98DHwCfADUAA8SbQkEKdaqTHwww/Uhv4CAuOWRBAHLRvHU6N1y1H1ajDSp42x4sunavz2fsUE6Qd+w5iG/AlUgHq4ApGygDiQBnJAB0JMId01J+0z5iVjaiw9FRlrAxL4jkV9NGCs4DI6UKU+NgK6APQCf4T0AEVgDHCAAEDwzxJhGDsMlAxJAxkgBSQASUN+2Fk1NBoPxw7gh6aaFv0fcXrCJAJx9M8AAAAASUVORK5CYII=',
      'searchUrl': 'https://hduse.net/search.php?action=do_search&postthread=1&forums=all&showresults=threads&keywords=%tt%'},`

Also Turkish torrent website Turkseed changed its url to trendfilm.net, I used find and replace to update it.

Has anyone managed to get the script working for [libremdb](https://github.com/zyachel/libremdb)? It still uses the same %tt% so could be a rather simple implementation. Plus the UI is simpler and cleaner.

**Comment by Purfview** [comment]
> Turkish torrent website Turkseed changed its url to trendfilm.net

Changed: https://github.com/Purfview/IMDb-Scout-Mod/commit/436ad4a704d1677ee39057def3646bf1c370de7f

> Paid Usenet forum HDUse can be searched with the below settings

I don't see 'matchRegex' in your post, so it can't be searched by the script, I would need an invite to add it myself.
So, post 'matchRegex' for it, that means when nothing is found.

> Has anyone managed to get the script working for [libremdb](https://github.com/zyachel/libremdb)?

I've no idea. I'm not interested in creating the scripts for the other sites.

**Reply by ArmanCevi** [reply]
Thanks for replying!

Here's with MatchRegex in all the supported languages of the forum. Also removed icon cause it was too small.

name: HDUse
searchUrl: https://hduse.net/search.php?action=do_search&postthread=1&forums=all&showresults=threads&keywords=%tt%
matchRegex: /keine Ergebnisse gefunden|no results were returned|No se encontraron resultados|aucun résultat n’a été trouvé|Nic nie znaleziono|нет результатов|sonuç bulunamadı/

Regarding libremd, it's basically a frontend for imdb itself, so the url structure is the same (e.g. we can extract the tt just the same). I would try out what works and what doesn't if I could figure out how to force the script to work on another domain

**Comment by Purfview** [comment]
Added HDUse -> https://github.com/Purfview/IMDb-Scout-Mod/commit/29da4af7762349a0b63790ce5e38689c9a9dd5f0 , I'll bump version number later.

>Regarding libremd, it's basically a frontend for imdb itself, so the url structure is the same

URL structure is not important, HTML structure is different and some crucial parts are missing.

**Reply by ArmanCevi** [reply]
Oh I see, thanks for enlightening me!
